firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -251,35 +253,39 @@ class TrackingProtectionTests: BaseTestCase {
         waitUntilPageLoad()
 
         if checkTrackingProtectionOn() {
-            mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection])
+            mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection], timeout: 5)
```

Comment:
The default `timeout` for `mozWaitForElementToExist` is 20. Why do we put a shorter timeout?

Reply:
When i used the default value the test still kept failing, this way i didn't have problems, i know it doesn't make sense, but not sure why.

Reply:
How about `TIMEOUT` or `TIMEOUT_LONG`?

Reply:
haven't try it, but visually i noticed that the element dont require more than 2 seconds to load

Reply:
I've approved the PR, but may think of a better way to mitigate the test flakiness. 🤷🏼‍♀️